Dockerfile

```
@@ -0,0 +1,12 @@
+FROM golang:alpine as builder
```

Mejor usar una versión concreta de Go (o almenos una versión mayor), ya que aunque no se suelen romper las build con los cambios de versión, no se puede asegurar al 
```suggestion
FROM golang:1.15-alpine as builder
```